proto/src/determined/api/v1/shell.proto

```diff
@@ -37,6 +37,9 @@ message GetShellsRequest {
   repeated string users = 5;
```

Comment:
comparing with https://github.com/determined-ai/determined/pull/5730/files?show-viewed-files=true&file-filters%5B%5D= looks like you're missing the sort parameter

Reply:
ah good catch